src/meltano/cli/job.py

```diff
@@ -46,12 +51,14 @@ def _list_single_job(
         click.echo(
             json.dumps({"job_name": task_set.name, "tasks": task_set.tasks}, indent=2)
         )
-    tracker = LegacyTracker(project)
-    tracker.track_meltano_job("list", job_name)
+    legacy_tracker = LegacyTracker(project)
```

Comment:
So `job` is a little bit different than config. Since `job` also uses the legacy tracker , you want initialize both the new style and legacy ones together:

https://github.com/meltano/meltano/blob/c41c2c153c00c0e01b8eabe5160429de6d6683b6/src/meltano/cli/install.py#L42-L43

This way you can seed the legacy tracker with the same starting contexts as the new tracker.

When you initialize a tracker `tracker = Tracker(project)` behind the scenes two contexts are immediately attached that describe the Project and Environment. Passing those to the legacy tracker by doing a `legacy_tracker = LegacyTracker(project, context_overrides=tracker.contexts) ` ensures that @pnadolny13 has shared context he can use to tie the two trackers together into a single session.